source/reference/kubectl-minio-plugin/kubectl-minio-init.rst

```diff
@@ -83,7 +83,7 @@ The command supports the following flags:
    :optional:
 
    The image to use when deploying the :minio-git:`MinIO Console <console>` in Operator mode, where administrators can create and manage MinIO tenants using a Graphical User Interface.
-   Defaults to ``minio/console:v0.17.3``.
+   Defaults to the `version bundled in variable DefaultOperatorImage for the matching Operator release <https://github.com/minio/operator/blob/master/kubectl-minio/cmd/helpers/constants.go>`.
```

Comment:
Needs a trailing double underscore `__` before the `.` at the end. Otherwise Sphinx won't make it a link.